pycytominer/cyto_utils/collate.py

```diff
@@ -0,0 +1,242 @@
+import os
+import subprocess
+import sys
+
+
+def run_check_errors(cmd):
+    """Run a system command, and exit if an error occurred, otherwise continue"""
+    if type(cmd) == str:
+        cmd = cmd.split()
+    output = subprocess.run(cmd, capture_output=True, text=True)
+    if output.stderr != "":
+        print_cmd = " ".join(map(str, cmd))
+        sys.exit(
+            f"The error {output.stderr} was generated when running {print_cmd}. Exiting."
+        )
+    return
+
+
+def collate(
+    batch,
+    config,
+    plate,
+    base_directory="../..",
+    column=None,
+    munge=False,
+    pipeline="analysis",
+    remote=None,
+    aggregate_only=False,
+    temp="/tmp",
+    overwrite=False,
+    add_image_features=True,
+    image_feature_categories=["Granularity", "Texture", "ImageQuality", "Threshold"],
+    printtoscreen=True,
+):
+    """Collate the CellProfiler-created CSVs into a single SQLite file by calling cytominer-database
+
+    Parameters
+    ----------
+    batch : str
+        Batch name to process
+    config : str
+        Config file to pass to cytominer-database
+    plate : str
+        Plate name to process
+    base_directory : str, default "../.."
+        Base directory where the CSV files will be located
+    column : str, optional, default None
+        An existing column to be explicitly copied to a Metadata_Plate column if Metadata_Plate was not set
+    munge : bool, default False
+        Whether munge should be passed to cytominer-database, if True will break a single object CSV down by objects
+    pipeline : str, default 'analysis'
+        A string used in path creation
+    remote : str, optional, default None
```

Comment:
```suggestion
    aws_remote_dir : str, optional, default None
```

If it's specific to AWS, we should explicitly note this

Reply:
I'll call it aws_remote but it's not really a "dir"